test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterDuplicateMovementReferenceNumberControllerSpec.scala

```diff
@@ -248,6 +248,24 @@ class EnterDuplicateMovementReferenceNumberControllerSpec
         ).value shouldBe routes.EnterImporterEoriNumberController.enterImporterEoriNumber().url
       }
 
+      "show the back button" in {
```

Comment:
Here is the problem:
You have added a back link in 4 different places, they could all be different, however you only test only one of those changes.
I would not be a big fan of writing 4 different tests for this simple thing, but rather change the code, so that you:
- either collapse those match - cases into a single one (this is to hard for you at the moment)
- val backLink = routes.SelectReasonForBasisAndClaimController.selectReasonForClaimAndBasisSubmit()
 and then use that backLink everywhere